src/main/java/org/ieeervce/gatekeeper/controller/SocietyController.java

```diff
@@ -0,0 +1,51 @@
+package org.ieeervce.gatekeeper.controller;
+
+import org.ieeervce.gatekeeper.ItemNotFoundException;
+import org.ieeervce.gatekeeper.dto.SocietyDTO;
+import org.ieeervce.gatekeeper.entity.Society;
+import org.ieeervce.gatekeeper.service.SocietyService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/society")
+public class SocietyController {
+    private final SocietyService societyService;
+    private final ModelMapper modelMapper;
+
+    @Autowired
+    public SocietyController(SocietyService societyService, ModelMapper modelMapper) {
+        this.societyService = societyService;
+        this.modelMapper = modelMapper;
+    }
+
+    @GetMapping
+    public List<Society> list(){
```

Comment:
Is there a standardized format for data return, such as ResultDTO?

Reply:
I thought of that, but decided to not since current API design so far isn't using it either.

In retrospect, it is much more standard to check the response code than the "ok" field.

Reply:
It would make much more sense to have consistent data return across all end point.